src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,85 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_SIZE = 16;
+    private static final double RESIZE_BOUND = 0.75;
+    private int size;
+    private int threshold;
+    private Node<K, V>[] nodeArray;
+
+    public MyHashMap() {
+        nodeArray = new Node[DEFAULT_SIZE];
```

Comment:
Use the diamond operator with the proper type parameters to ensure type safety. It should be 'new Node<K,V>[DEFAULT_SIZE];'. However, Java doesn't allow the creation of generic arrays directly. You can use 'new Node[DEFAULT_SIZE];' and add a SuppressWarnings annotation or consider using an alternative approach like an ArrayList or casting a Node array.